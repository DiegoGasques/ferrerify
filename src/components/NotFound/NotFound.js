import React from "react";
import { Link } from "react-router-dom";

export const NotFound = () => (
  <div className="NotFound">
    404 - <Link to="/">Go home</Link>
  </div>
);

export default NotFound;
