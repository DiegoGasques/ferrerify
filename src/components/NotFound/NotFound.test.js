import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { NotFound } from "./NotFound";

describe("[COMPONENT] NotFound", () => {
  it("Tests for changes in the rendered component", () => {
    const wrapper = shallow(<NotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
