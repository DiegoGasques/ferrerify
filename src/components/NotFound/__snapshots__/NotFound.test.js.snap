// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] NotFound Tests for changes in the rendered component 1`] = `
<div
  className="NotFound"
>
  404 - 
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
